pkgs/by-name/ta/taguette/package.nix

```diff
@@ -0,0 +1,89 @@
+{
+  lib,
+  stdenvNoCC,
```

Comment:
Nit: you're not using it to build anything, as a matter of style I'd use `stdenv` for this.